components/lcd/esp_lcd_ssd1681/esp_lcd_panel_ssd1681.c

```diff
@@ -0,0 +1,654 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "sdkconfig.h"
+#if CONFIG_LCD_ENABLE_DEBUG_LOG
+// The local log level must be defined before including esp_log.h
+// Set the maximum log level for this source file
+#define LOG_LOCAL_LEVEL ESP_LOG_DEBUG
+#endif
+#include "esp_log.h"
+#include "esp_check.h"
+#include "esp_memory_utils.h"
+#include "esp_attr.h"
+#include "driver/gpio.h"
+#include "esp_lcd_panel_ssd1681.h"
+#include "esp_lcd_panel_interface.h"
+#include "esp_lcd_panel_io.h"
+#include "esp_lcd_panel_vendor.h"
+#include "esp_lcd_ssd1681_commands.h"
+
+#define SSD1681_LUT_SIZE                   159
+#define SSD1681_EPD_1IN54_V2_WIDTH         200
+#define SSD1681_EPD_1IN54_V2_HEIGHT        200
```

Comment:
The maximum resolution that SSD1681 supports is 200x200. For one hand, I have never seen an ssd1681 e-paper panel with resolution smaller than 200x200, for the other hand, even if there do exist, we could just simply set the begin and end coordinate accordingly. Fixing the resolution to 200x200 does not waste too much memory since it is monochrome.

Reply:
Ok, we can keep it as it is.